tests/__main__.py

```diff
@@ -87,7 +94,9 @@ def convert_argv(argv):
         args = cmd + args
     else:
         args = env = None
-    return args, env, runtests, lint
+
+    RunArgs = namedtuple('RunArgs', 'argv, env, junit_xml_file')
```

Comment:
A few things to consider:

* typically class definitions belong at module-global level
* I find it useful to give the class a bit more description and room to grow:

    class RunArgs(namedtuple('RunArgs', 'argv env junit_xml_file')):
        """The arguments to run_tests()."""

* note the omitted the commas in the fields string
* all that said, using a namedtuple for a one-off situation like this might be more trouble than it's worth; I'd say just return a normal tuple and be done with it:

    runargs = (argv, env, junit_xml_file, coverage) if runtests else None
    return runargs, lint

Also, note that I factored ``coverage`` out of ``runtests``, meaning both can be strictly boolean.

Reply:
Ah I see. I didn't fully grok the namedtuple usage, and that makes sense. (C# Dictionary is my go-to, I was looking for a parallel here). And I get what you are going for then - I will switch that around.